model/atmosphere/diffusion/tests/diffusion_stencil_tests/test_apply_diffusion_to_vn.py

```diff
@@ -89,15 +90,14 @@ def reference(
                 vn,
             )
 
+        # restriction of execution domain
```

Comment:
Use `helpers.Output`  with slices to restrict comparison domain?

Reply:
agreed to leave as is for now